app/assets/javascripts/school_administrator_dashboard/discipline_dashboard/SchoolDisciplineDashboard.js

```diff
@@ -117,23 +160,29 @@ export default class SchoolDisciplineDashboard extends React.Component {
     const {timeRangeKey} = this.state;
     const {school} = this.props;
     const chartOptions = [
-      {value: 'location', label: 'Location'},
-      {value: 'time', label: 'Time'},
+      {value: 'incident_location', label: 'Location'},
+      {value: 'exactTime', label: 'Time'},
```

Comment:
let's keep the case for keys the same within the same data structure :)

Reply:
Oops! Yeah, I was experimenting and left one of these in.